src/Microsoft.TestPlatform.ObjectModel/DataCollector/IDataCollectorAttachmentProcessor.cs

```diff
@@ -22,6 +22,18 @@ public interface IDataCollectorAttachmentProcessor
 
         /// <summary>
         /// Indicates whether attachment processor is supporting incremental processing of attachments
+        /// 
+        /// `SupportsIncrementalProcessing` should indicate if attachment processor is supporting incremental processing of attachments. It means that `ProcessAttachmentSetsAsync` should be [associative](https://en.wikipedia.org/wiki/Associative_property).
```

Comment:
Consider putting detailed explanation in the <remarks> section, so intellisense is manageable.

Reply:
fixed